.github/install_dependency_branch.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Define variables
+dependency_name=$1
+branch_name=$2
+
+root_dir=$(pwd)
+dependency_branch_name='main'
+branch_name_to_check=${branch_name}
+iterations=0
+
+while [ "$branch_name_to_check" != "$dependency_branch_name" ] && [ $iterations -lt 20 ]
```

Comment:
Why again do we need to loop?

Reply:
+ 1 this question

Reply:
Imagine you have branch `foo` which branches off of `bar` which branches off of `baz`, and layered config tree only has branch `baz`. We want to check out branch `baz` on layered config tree.